woocommerce/woocommerce
# Remove duplicated CSS content

Ref: #27410

### All Submissions:






### Changes proposed in this Pull Request:



This is a fix for a problem that probably happened due to some kind of error and caused the 68 repetitions of the minified file to replace the unminified CSS version.

Theoretically, we might drop unminified resources as mentioned in the issue #27410, but that's an exercise for another day. This is low hanging fruit that just fixes a problem and (theoretically speaking) should not cause any problems whatsoever.
It seems like those files are not even used anywhere in our codebase (only the minified version is), but I think we can't just remove them due to backward compatibility reasons.

The initial hope was to reduce the zip file size, but I think due to the nature how zip algorithm works, the effect on the zip file size is negligible (if not negative, as 4.6-beta.1 size is 8,810,294 bytes while the result on master (using `npm run build:zip`) with this change is 8,992,046 bytes).

### How to test the changes in this Pull Request:

1. Apply branch
2. Check that areas using the jquery-ui CSS still work fine, e.g. date picker in Order edit screen:
![Screenshot 2020-09-24 at 11 14 32](https://user-images.githubusercontent.com/2207451/94125942-503ce780-fe57-11ea-9704-c6e8d209bc74.png)

### Other information:




### Changelog entry

> Fix - Removed duplicated CSS code from jQuery UI.
